core/src/main/java/com/linecorp/armeria/internal/logging/CountingSampler.java

```diff
@@ -46,20 +48,20 @@
  *
  * <p>Forked from brave-core 5.6.3 at d4cbd86e1df75687339da6ec2964d42ab3a8cf14
  */
-final class CountingSampler extends Sampler {
+public final class CountingSampler implements Sampler {
```

Comment:
Could become package-private again because a user can create it via `Sampler.random()`.